docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -51,6 +54,7 @@ Example:
 - `description : String`: Optional description of the template. Shown in the element template selection modal and in the properties panel (after applying an element template).
 - `documentationRef : String`: Optional URL pointing to a template documentation. Shown in the properties panel (after applying an element template).
 - `version : Integer`: Optional version of the template. If you add a version to a template, it is considered unique based on its ID and version. Two templates can have the same ID if their version is different.
+- `engines : Object`: Optional dictionary of execution platforms compatible with the template. Execution platform version is specified with semantic versions range.
```

Comment:
Execution platform -> still too specific

Reply:
Is this solely for this description, or applicable to all references of execution platform?

Reply:
"Execution platform" is a term for the runtimes: Camunda 7, Camunda 8.
The engines field can contain a name of anything that the template may be compatible/incompatible with. Built-in are the checks for:
* execution platform
* application (Web/Desktop Modeler)
* element templates runtime
Still, I think if we later in the doc point the users to _applications and execution platforms_, it is still fine. I wouldn't expect a user who creates templates to be aware of the element templates runtime version, and it is also covered by the application version because it uses a specific element templates runtime.